Technical Advisories (#21537)
yugabyte/yugabyte-db: docs/content/preview/releases/techadvisories/_index.md

---
title: YugabyteDB Technical Advisories
headerTitle: Technical Advisories
linkTitle: Technical Advisories
description: List of technical advisories
image: /icons/triangle-exclamation-thin.svg
type: indexpage
showRightNav: true
cascade:
  unversioned: true
---

> Here you will find important notifications and updates regarding product issues, security vulnerabilities, and other technical matters that may impact our products and services. We understand that keeping your systems secure and running smoothly is of utmost importance. This page serves as a central resource to ensure you have access to the latest technical advisories issued by our team.

Reviewer: ```suggestion
Technical Advisories include important notifications and updates regarding product issues, security vulnerabilities, and other technical matters that may impact Yugabyte products and services. We understand that keeping your systems secure and running smoothly is of utmost importance. 
This page serves as a central resource to ensure you have access to the latest technical advisories issued by Yugabyte.
```

We strongly recommend regularly reviewing this page and taking appropriate measures outlined in the advisories to mitigate potential risks and disruptions. Our team continuously monitors for emerging threats and technical concerns. New advisories will be posted here as soon as they become available.

## List of advisories

| Advisory&nbsp;Number |    Synopsis     | Product  | Affected Versions | Publish&nbsp;Date |
| :------- | :-------------- | :------- | :---------------- | :----------- |

> | {{<ta 21297>}}| Missing writes during Batch Execution in a transaction | [YSQL](./) | [All](/preview/releases/ybdb-releases/)            | Mar-25-2024  |

Reviewer: ```suggestion
| {{<ta 21297>}}| Missing writes during Batch Execution in a transaction | [YSQL](../../api/ysql/) | [All](../ybdb-releases/)            | March 25, 2024  |
```

Here you will find important notifications and updates regarding product issues, security vulnerabilities, and other technical matters that may impact our products and services. We understand that keeping your systems secure and running smoothly is of utmost importance. This page serves as a central resource to ensure you have access to the latest technical advisories issued by our team.

> We strongly recommend regularly reviewing this page and taking appropriate measures outlined in the advisories to mitigate potential risks and disruptions. Our team continuously monitors for emerging threats and technical concerns. New advisories will be posted here as soon as they become available.

Reviewer: ```suggestion
It is strongly recommended that you regularly review this page and take appropriate measures outlined in the advisories to mitigate potential risks and disruptions. Yugabyte continuously monitors for emerging threats and technical concerns.
```
